src/metadata.cc

```diff
@@ -246,6 +250,17 @@ class MetadataWorker : public Napi::AsyncWorker {
           Napi::Buffer<char>::NewOrCopy(env, baton->tifftagPhotoshop,
             baton->tifftagPhotoshopLength, sharp::FreeCallback));
       }
+      if (baton->comments.size() > 0) {
+        int i = 0;
+        Napi::Array comments = Napi::Array::New(env);
```

Comment:
It looks like we know the number of comments here so could pre-size the array, something like:
```suggestion
        Napi::Array comments = Napi::Array::New(env, baton->comments.size());
```